### ISSUE TITLE
Elevated urine acetoacetic acid concentrations

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6016]
**Preferred term label:** Elevated urine acetoacetic acid concentrations

**Synonyms** Elevated urine diacetic acid concentrations

**Definition (free text, please give PubMed ID)** Concentration of acetoacetic acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Ketonuria HP:0002919

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
